pfurl/pfurl.py

```diff
@@ -982,6 +965,7 @@ def push_core(self, d_msg, **kwargs):
 
         return d_ret
 
+#This is not called by anything...... ???????
```

Comment:
Formatting is wrong.

Reply:
oops didnt mean to leave that comment there. ill remove it